Src/FluentAssertions/FluentAssertions.csproj

```diff
@@ -56,6 +56,9 @@
   <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.0'">
     <PackageReference Include="System.Configuration.ConfigurationManager" Version="4.4.0" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
```

Comment:
We're no longer targeting net core 3.0

Reply:
But we do support it officially. So I would keep it in.

Reply:
Isn't this superfluous if we aren't targeting net core 3.0?

Reply:
Correct.

Reply:
My response was meant for your comment on the nuspec file.